libs/ha/ha_common_libs_k8s.py

```diff
@@ -925,14 +925,11 @@ def event_s3_operation(self, event, setup_s3bench=True, log_prefix=None, s3useri
                 skip_write=skipwrite, skip_read=skipread, obj_size=workload,
                 skip_cleanup=skipcleanup, log_file_prefix=f"log_{log_prefix}",
                 end_point=S3_CFG["s3_url"], validate_certs=S3_CFG["validate_certs"])
-            if event.is_set():
+            if event.is_set() or (isinstance(event_set_clr, list) and event_set_clr[0]):
```

Comment:
Instead of checking type list here, make its default value as 'event_set_clr : list = None'.

Reply:
This will result in event_set_clr : list = None, if no value passed to this variable

Reply:
if it is not list then do we want to go to else part?
I guess no need to check list type.
What do you think?

Reply:
As, will keep as it is as this change is required where we are not sending any value to event_set_clr